states/new_jersey/elections.toml

```diff
@@ -3,7 +3,140 @@ date = 2020-07-07
 name = "Congressional & Presidential Primary"
 original_date = 2020-07-07
 
+[20200707.registration]
+received_by = 2020-06-16
+sources = ["https://www.state.nj.us/state/elections/voter-registration.shtml",
+           "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-02-primary-election-052220.pdf"]
+
+[20200707.absentee.application]
+received_by = 2020-06-30
+in_person_by = 2020-07-07T20:00:00
+sources = [
+    "https://www.state.nj.us/state/elections/primary-election-070720.shtml",
+    "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-02-primary-election-052220.pdf"
+]
+
+[20200707.absentee.military_overseas.application]
+online_by = "2020-07-03"
+sources = ["https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-02-primary-election-052220.pdf"]
+
+[20200707.absentee]
+postmarked_by = 2020-07-07
+received_by = 2020-07-14T20:00:00
+in_person_by = 2020-07-07T20:00:00
+sources = ["https://www.state.nj.us/state/elections/assets/pdf/election-results/2020/2020-drop-box-locations.pdf",
+           "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-02-primary-election-052220.pdf"]
+
+[20200707.poll]
+in_person_start = 2020-07-07T06:00:00
+in_person_by = 2020-07-07T20:00:00
+sources = ["https://elections.delaware.gov/calendar/2020ElectionCalendar.pdf",
+           "https://lis.njleg.state.nj.us/nxt/gateway.dll/statutes%2F1%2F19539%2F19749"]
+
+[20200929]
+date = 2020-09-29
+name = "September Special School Board"
+original_date = 2020-09-29
+
+[20200929.registration]
+received_by = 2020-09-08
+in_person_by = 2020-09-08
+sources = ["https://www.state.nj.us/state/elections/voter-registration.shtml",
+           "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-08-september-school-board.pdf"]
+
+[20200929.absentee.application]
+received_by = 2020-09-29
+in_person_by = 2020-09-28T15:00:00
+sources = [
+    "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-08-september-school-board.pdf"
+]
+
+[20200929.absentee.military_overseas.application]
+online_by = 2020-09-25
```

Comment:
I was also thinking we might need online_by for .registration its frequently available online as well. I don't recall it being different than received_by,  but we may want to always include online_by and received_by if there are two methods and the same date... also feels like extra work, but it also more accurate. Thoughts ?

Reply:
Oh ya, that would be good.

Reply:
When multiple methods are doable they should have separate entries, even if the dates are the same. That way we'll generate reminders for each.